packages/template-retail-react-app/app/components/unavailable-product-confirmation-modal/index.test.js

```diff
@@ -10,349 +10,86 @@ import UnavailableProductConfirmationModal from '@salesforce/retail-react-app/ap
-                    }
-                },
-                {
-                    orderable: true,
-                    price: 61.43,
-                    productId: '701642889854M',
-                    variationValues: {
-                        color: 'JJ0NLD0',
-                        size: '9XL'
-                    }
-                }
-            ],
-            variationAttributes: [
-                {
-                    id: 'color',
-                    name: 'Colour',
-                    values: [
-                        {
-                            name: 'Laurel',
-                            orderable: true,
-                            value: 'JJ0NLD0'
-                        }
-                    ]
-                },
-                {
-                    id: 'size',
-                    name: 'Size',
-                    values: [
-                        {
-                            name: 'S',
-                            orderable: true,
-                            value: '9SM'
-                        },
-                        {
-                            name: 'M',
-                            orderable: true,
-                            value: '9MD'
-                        },
-                        {
-                            name: 'L',
-                            orderable: true,
-                            value: '9LG'
-                        },
-                        {
-                            name: 'XL',
-                            orderable: true,
-                            value: '9XL'
-                        }
-                    ]
-                }
-            ],
-            variationValues: {
-                color: 'JJ0NLD0',
-                size: '9MD'
-            },
-            c_color: 'JJ0NLD0',
-            c_refinementColor: 'black',
-            c_size: '9MD',
-            c_width: 'Z'
+            }
+        },
+        {
+            currency: 'GBP',
+            id: '701642889829M',
+            imageGroups: [],
+            inventory: {
+                ats: 68,
+                backorderable: false,
+                id: 'inventory_m',
+                orderable: false,
+                preorderable: false,
+                stockLevel: 0
+            }
         }
     ]
 }
 
 describe('UnavailableProductConfirmationModal', () => {
-    beforeEach(() => {
+    test('renders without crashing', () => {
         prependHandlersToServer([
             {
                 path: '*/products',
                 res: () => {
-                    return mockProducts
+                    return mockProductsWithUnavailableProducts
                 }
             }
         ])
-    })
-    test('renders without crashing', () => {
         expect(() => {
             renderWithProviders(<UnavailableProductConfirmationModal />)
         }).not.toThrow()
     })
 
     test('opens confirmation modal when unavailable products are found', async () => {
```

Comment:
How about having more than 1 test to verify handling of the unavailable products? 

It looks like currently we're testing only the case when `orderable=false`. But our implementation looks for other cases too. Can we test all the other scenarios please?